src/algorithms/tonal/chordsdetectionbeats.cpp

```diff
@@ -48,7 +48,7 @@ void ChordsDetectionBeats::configure() {
   _sampleRate = parameter("sampleRate").toReal();
   _hopSize = parameter("hopSize").toInt();
   _chromaPick = parameter("chromaPick").toLower();
-  if (!(_chromaPick == "interbeat_median" || _chromaPick == "first_beat"))
```

Comment:
Need to update the parameter in the header file too.

Reply:
Ok, done.